flutter/test/integrations/flutter_error_integration_test.dart

```diff
@@ -23,7 +23,7 @@ void main() {
     }) {
       // replace default error otherwise it fails on testing
       FlutterError.onError =
-          handler ?? (FlutterErrorDetails errorDetails) async {};
+          handler ??= (FlutterErrorDetails errorDetails) async {};
```

Comment:
This would only replace if theres nothing in there, which is different than the comment above